source/administration/upgrading-to-2.0.rst

```diff
@@ -0,0 +1,82 @@
+Upgrading to version 2.0
+========================
+
+To upgrade an early edition of Mattermost to the latest edition, you must first upgrade in a specific sequence until you get to version 2.0.0. After that, you must upgrade to version 3.0.3, and from there you can upgrade directly to the latest Mattermost version.
```

Comment:
Propose putting something here about which server versions this guide is for. And if they have a different server version than that mentioned, which guides w=should they be using instead. It makes me nervous to have a lot of different guides without explicitly stating what each are for

- [ ]